dev/scripts/build.ts

```diff
@@ -3,46 +3,139 @@ import { join } from "node:path"
 import { readJson, writeJson } from "../attest/src/fs.js"
 import { shell } from "../attest/src/shell.js"
 import { repoDirs } from "./common.js"
+import { rewritePaths } from "./overwrite.js"
+import type { ReplacementDictionary } from "./overwrite.js"
 
 const packageRoot = process.cwd()
 const outRoot = join(packageRoot, "dist")
 const packageJson = readJson(join(packageRoot, "package.json"))
+const tempTsConfigBaseName = "tsconfig.temp"
+const tempTsConfigPath = join(packageRoot, `${tempTsConfigBaseName}.json`)
 
-const buildFormat = (module: "commonjs" | "esnext") => {
-    const outDir = join(outRoot, module === "commonjs" ? "cjs" : "mjs")
-    writeJson(tempTsConfig, {
-        ...tsConfigData,
-        include: ["src"],
+const nuke = (target: string) =>
```

Comment:
Could we rename these to `rmRf` and `cpR` and move them to attest/src/fs?

Those names would give me a better intuition for what to expect, although I could also see a case being made for just having a constant object called `recursiveForce` or something, then making the calls to `rmSync`/`cpSync` directly and passing that to make it more obvious what's going on.

I've moved away a bit from creating utilities that are really thin wrappers around builtins as they often end up being forgotten and unused, although I would totally do something like this here to avoid typing `{ recursive: true, force: true }` everywhere